src/gui/DatabaseOpenWidget.cpp

```diff
@@ -105,8 +105,7 @@ void DatabaseOpenWidget::showEvent(QShowEvent* event)
 #ifdef WITH_XC_YUBIKEY
     // showEvent() may be called twice, so make sure we are only polling once
     if (!m_yubiKeyBeingPolled) {
-        connect(
-            YubiKey::instance(), SIGNAL(detected(int,bool)), SLOT(yubikeyDetected(int,bool)), Qt::QueuedConnection);
+        connect(YubiKey::instance(), SIGNAL(detected(int, bool)), SLOT(yubikeyDetected(int, bool)), Qt::QueuedConnection);
```

Comment:
Remove spaces from within SIGNAL and SLOT

Reply:
Why? Is that standard somewhere? Really ugly also when the signature is const QString& or so.

Reply:
Sorry I meant at the commas within the signal/slot areas. The first refactor PR did this.

Reply:
You mean ` SIGNAL(detected(int,bool))` instead of ` SIGNAL(detected(int, bool))`, right? Looks rather cringeworthy if you ask me.

Reply:
Uh, that's new to me. I don't like how it looks, but I guess I'll change it then.